unit_testing/myapp/migrations/0002_auto__add_field_person_email.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import SchemaMigration
+from django.db import models
+
+
+class Migration(SchemaMigration):
+
+    def forwards(self, orm):
+        # Adding field 'Person.email'
+        db.add_column(u'myapp_person', 'email',
+                      self.gf('django.db.models.fields.EmailField')(default='hello@hello.com', max_length=75),
```

Comment:
No default value is set in the model, however south requires that a default value be provided in order to update any existing entries when a field is added and migrations are performed. This is only that value.